### PR TITLE
[11.0] German toponyms

### PR DESCRIPTION
**Pull request by fkantelberg**
This module should add the zip codes and toponyms for Germany using the same approach as the spanish version.

It depends on the OCA/partner-contact repository for the wizard to download from GeoNames.

### CODE REVIEW

**Comment by kmatthes** [comment]
@oca-clabot Since @fkantelberg's contribution was done as part of his job by initOS and initOS already signed an Entity CLA, I just sent an request to add @fkantelberg to cla AT odoo-community.org.

**Comment by OSevangelist** [comment]
@pedrobaeza i think this one is alright now an can be merged. the holiday stuff we recently spoke about is not part of it anymore right.

**Comment by kmatthes** [comment]
FYI: Rebecca informed me, that @fkantelberg was added to initOS and @ITISAG-OCA was added as contact under IT IS AG.

**Comment by OSevangelist** [comment]
@pedrobaeza would you please be so kind to merge this one. I thing we shall start with the v12 migration right after this

File: l10n_de_toponyms/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 IT IS AG <oca@itis.de>
+# Copyright 2018 Florian Kantelberg <florian.kantelberg@initos.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "German Toponyms",
+    "version": "11.0.1.0.0",
+    "author": '''IT IS AG Germany,
```
Comment (pedrobaeza) [inline l10n_de_toponyms/__manifest__.py]: Please don't use triple quote, as this adds unneeded spaces between each name

File: l10n_de_toponyms/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "category": "Localization",
+    "depends": [
+        "base_location_geonames_import",
+        "l10n_de_country_states",
+    ],
+    "license": "AGPL-3",
+    "data": [
+        "wizard/l10n_de_toponyms_wizard.xml",
+    ],
+    'images': [
```
Comment (pedrobaeza) [inline l10n_de_toponyms/__manifest__.py]: Don't add empty keys

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: l10n_de_toponyms/__manifest__.py (hunk 1 above)
Comment (OSevangelist) [reply l10n_de_toponyms/__manifest__.py]: @pedrobaeza damned we will correct that in a minute ;-) @fkantelberg you could you please give quick hands here

**Review by pedrobaeza — APPROVED** [review]

**Comment by OSevangelist** [comment]
@pedrobaeza thx !

**Pull request merged**